client/db_test.go

```diff
@@ -245,3 +245,29 @@ func ExampleDB_Insecure() {
 	// Output:
 	// aa=1
 }
+
+func ExampleDB_Open() {
```

Comment:
Oh, that's subtle. I didn't realize that `ExampleDB_Insecure` wasn't showing up in the docs either (do examples that don't fit the pattern even get run?). I'd say that both of them should be converted to non-example tests, then, but it can be done in a separate change. It's fine to just do a simple test here to make sure that Open doesn't return an error or panic.